tests/rkt_api_service_test.go

```diff
@@ -230,7 +271,37 @@ func TestAPIServiceListInspectPods(t *testing.T) {
 		t.Errorf("Unexpected result: %v, should see zero pods", resp.Pods)
 	}
 
-	patchImportAndRun("rkt-inspect-print.aci", []string{"--exec=/inspect --print-msg=HELLO_API --exit-code=42"}, t, ctx)
+	patches := []string{"--exec=/inspect --print-msg=HELLO_API --exit-code=42"}
+	patchImportAndFetchHash("rkt-inspect-print.aci", patches, t, ctx)
+	iresp, err := c.ListImages(context.Background(), &v1alpha.ListImagesRequest{})
```

Comment:
Can we move this logic (testing the annotations in image) to the `TestAPIServiceListInspectPods`?